app/uk/gov/hmrc/voapropertylinking/models/modernised/agentrepresentation/AppointmentChangesRequest.scala

```diff
@@ -37,4 +37,12 @@ object AppointmentChangesRequest {
       scope = AppointmentScope.withName(appointAgent.scope),
       propertyLinks = None
     )
+
+  def apply(unassignAgent: UnassignAgent): AppointmentChangesRequest =
+    AppointmentChangesRequest(
+      agentRepresentativeCode = unassignAgent.agentRepresentativeCode,
+      action = AppointmentAction.REVOKE,
+      scope = AppointmentScope.withName(unassignAgent.scope),
+      propertyLinks = None
+    )
```

Comment:
Is this used for only unassign agent from *all* property links?

Reply:
yes, it will be used by that. It will also be used for removing agent from organisation